src/common/pa_converters.c

```diff
@@ -358,18 +383,21 @@ static void Float32_To_Int32_Dither(
 {
     float *src = (float*)sourceBuffer;
     PaInt32 *dest =  (PaInt32*)destinationBuffer;
+    int prevMode = SetRoundingMode();
 
     while( count-- )
     {
         /* REVIEW */
         double dither  = PaUtil_GenerateFloatTriangularDither( ditherGenerator );
         /* use smaller scaler to prevent overflow when we add the dither */
         double dithered = ((double)*src * (2147483646.0)) + dither;
```

Comment:
Now that we are rounding to nearest, will this smaller scaler be small enough to prevent numeric overflow during the cast?

Reply:
This PR does not change rounding mode because rounding to nearest is the default rounding mode when process starts and therefore de facto PA is already assuming this rounding mode. Proposed change enforces this rounding mode explicitly to avoid situation that user sets for example `FE_DOWNWARD` which actually changes rounding mode and thus the result of PA converters will be different if we do not set `FE_TONEAREST` explicitly.